.gitignore

```diff
@@ -1,8 +1,10 @@
 kbtui
 emojiList.go
+kbtui.toml
```

Comment:
You should probably remove this

Reply:
Resolved - i don't usually see config files included in git, but it seems to be what people want in favor of a defconfig!